tests/app/main/views/test_agreement.py

```diff
@@ -113,33 +102,26 @@ def test_unknown_gps_and_trusts_are_redirected(
     )
 
 
-@pytest.mark.parametrize('crown, expected_status, expected_file_fetched, expected_file_served', (
-    # (
-    #     True, 200, 'crown.pdf',
-    #     'U.S. Notify data sharing and financial agreement.pdf',
-    # ),
-    # (
-    #     False, 200, 'non-crown.pdf',
-    #     'U.S. Notify data sharing and financial agreement (non-crown).pdf',
-    # ),
+@pytest.mark.parametrize('expected_status, expected_file_fetched, expected_file_served', (
     (
-        None, 404, None,
-        None,
+        200, 'agreement.pdf',
+        'U.S. Notify data sharing and financial agreement.pdf',
     ),
 ))
+@mock.patch('app.s3_client.s3_mou_client.current_app')
 def test_download_service_agreement(
+    mock_flask_current_app,
     client_request,
     mocker,
     mock_get_service_organisation,
-    crown,
     expected_status,
     expected_file_fetched,
     expected_file_served,
 ):
+    mock_flask_current_app.config['MOU_BUCKET_NAME'] = 'test-mou'
```

Comment:
[issue] The MOU_BUCKET_NAME is not a thing that exists in live configs. What path might blow up from expecting it to exist?

Reply:
I'm a bit mystified by MOU_BUCKET_NAME at the moment.  The s3_mou_client clearly expects it to exist.

Reply:
Yeah, there used to be a bucket that held only 2 pdfs (crown and non-crown). Seemed like overkill, and this is probably something we can address in a separate issue/PR as long as the current signup and/or go-live path isn't any more broken than it used to be.

Reply:
I'm not sure how to test the signup path (since I've already signed up and you need a real gov email) or the go live path.   Right now I can only confirm that sending texts isn't broken.

Reply:
on your local machine, you can always blow away your db to start over: `dropdb notification_api; make bootstrap` 

of course, that will completely delete the db, so use with caution.

Reply:
This is functionality that will be removed, so I'm cool with however we want to do this for now.